hail/src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -88,6 +88,7 @@ object Literal {
 final case class Literal(_typ: Type, value: Annotation) extends IR {
   require(!CanEmit(_typ))
   require(value != null)
+  // require(SafeRow.isSafe(value))
```

Comment:
I see the main fix here about creating a `SafeRow` instead of an `UnsafeRow`. I don't see why this and the `assert` in `RVDPartitioner` are now commented out. If you intend to leave them commented out, I think I'd want a line comment above explaining why they are currently commented out.

Reply:
These are pretty expensive checks to be doing every time the node is constructed, I think that's why. But agree that a comment would be good.

Reply:
Ah, I read the diff wrong, thought these were things we previously did that are now commented out. Now see that they are newly added lines altogether. Agree that it would be an expensive check. I think I would still add a comment like "For debugging purposes" or something above the lines.